blocks/gu_spoverview/block_gu_spoverview.php

```diff
@@ -81,6 +81,7 @@ public function get_content() {
             }
         }
 
+
```

Comment:
Please remove this empty line. This file should not be included in pull request.